From 55f3ef80384165a4d4a3a8da8a4022110206dc79 Mon Sep 17 00:00:00 2001
From: Paul Chote <pchote@users.noreply.github.com>
Date: Mon, 6 May 2019 11:28:45 +0100
Subject: [PATCH 1/2] Fix Makefile targets

---
 Makefile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 55887761d87..1f4ab7e7260 100644
--- a/Makefile
+++ b/Makefile
@@ -79,7 +79,6 @@ INSTALL_DATA = $(INSTALL) -m644
 MSBUILD = msbuild -verbosity:m -nologo
 
 # program targets
-CORE = pdefault game utility server
 VERSION     = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
 
 # dependencies
@@ -213,7 +212,7 @@ version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mo
 		rm $${i}.tmp; \
 	done
 
-man-page: utility mods
+man-page: core
 	@mono --debug OpenRA.Utility.exe all --man-page > openra.6
 
 install: dependencies core install-core
@@ -395,4 +394,4 @@ help:
 
 .SUFFIXES:
 
-.PHONY: core package all mods clean distclean dependencies version nunit
+.PHONY: check-scripts check nunit test all core clean distclean cli-dependencies linux-dependencies linux-native-dependencies windows-dependencies osx-dependencies geoip-dependencies dependencies all-dependencies version man-page install install-linux-shortcuts install-engine install-common-mod-files install-default-mods install-core install-linux-icons install-linux-desktop install-linux-mime install-linux-appdata install-man-page install-linux-scripts uninstall help

From beff707043bcee9846d46863513b6d8a4471c026 Mon Sep 17 00:00:00 2001
From: Paul Chote <pchote@users.noreply.github.com>
Date: Mon, 6 May 2019 11:30:27 +0100
Subject: [PATCH 2/2] Merge man-page target into install-man-page

---
 Makefile | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 1f4ab7e7260..9e52d1ec4a9 100644
--- a/Makefile
+++ b/Makefile
@@ -212,9 +212,6 @@ version: VERSION mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mo
 		rm $${i}.tmp; \
 	done
 
-man-page: core
-	@mono --debug OpenRA.Utility.exe all --man-page > openra.6
-
 install: dependencies core install-core
 
 install-linux-shortcuts: install-linux-scripts install-linux-icons install-linux-desktop
@@ -304,9 +301,11 @@ install-linux-appdata:
 	@$(INSTALL_DATA) packaging/linux/openra-d2k.appdata.xml "$(DESTDIR)$(datadir)/appdata/"
 	@-$(RM) packaging/linux/openra-ra.appdata.xml packaging/linux/openra-cnc.appdata.xml packaging/linux/openra-d2k.appdata.xml
 
-install-man-page: man-page
+install-man-page:
 	@$(INSTALL_DIR) "$(DESTDIR)$(mandir)/man6/"
+	@mono --debug OpenRA.Utility.exe all --man-page > openra.6
 	@$(INSTALL_DATA) openra.6 "$(DESTDIR)$(mandir)/man6/"
+	@-$(RM) openra.6
 
 install-linux-scripts:
 ifeq ($(DEBUG), $(filter $(DEBUG),false no n off 0))
@@ -394,4 +393,4 @@ help:
 
 .SUFFIXES:
 
-.PHONY: check-scripts check nunit test all core clean distclean cli-dependencies linux-dependencies linux-native-dependencies windows-dependencies osx-dependencies geoip-dependencies dependencies all-dependencies version man-page install install-linux-shortcuts install-engine install-common-mod-files install-default-mods install-core install-linux-icons install-linux-desktop install-linux-mime install-linux-appdata install-man-page install-linux-scripts uninstall help
+.PHONY: check-scripts check nunit test all core clean distclean cli-dependencies linux-dependencies linux-native-dependencies windows-dependencies osx-dependencies geoip-dependencies dependencies all-dependencies version install install-linux-shortcuts install-engine install-common-mod-files install-default-mods install-core install-linux-icons install-linux-desktop install-linux-mime install-linux-appdata install-man-page install-linux-scripts uninstall help
